src/main/webapp/user-page.html

```diff
@@ -75,11 +75,16 @@ <h1 class="site-heading text-center text-white d-none d-lg-block">
               <h2 class="section-heading mb-4">
                 <span class="section-heading-lower">About Me</span>
               </h2>
+              <div class="with-events text-center" id="nickname-container">
+                <p class="text-center" contenteditable="true" id="nickname-text-logged-in">Loading...</p>
+                <button type="button" class="btn btn-warning" id="nickname-save-button" onclick="setNickname()">Save</button>
```

Comment:
Maybe change the text on all the buttons on this page (since there are more now) to indicate what they do? Like maybe "Save nickname" or "Save about me" instead of just "Save" or something along those lines?

Reply:
ah okay!